samcli/commands/build/build_context.py

```diff
@@ -508,3 +509,29 @@ def _is_layer_buildable(layer: LayerVersion):
             LOG.debug("Skip building pre-built layer: %s", layer.full_path)
             return False
         return True
+
+    _JAVA_BUILD_WARNING_MESSAGE = (
+        "Test the latest build changes for Java runtime 'SAM_CLI_BETA_MAVEN_SCOPE_AND_LAYER=1 sam build'. "
+        "These changes will replace the existing flow on 1st of April 2022. "
+        "Check __issue_line__ for more information."
```

Comment:
What's the `__issue_line__` referring to?

Reply:
We will update this warning message once we have an announcement GH issue ready in SAM CLI. This is just a placeholder for that